--- origsrc/wyrd-1.4.4/Makefile.in	2008-02-21 22:25:24.000000000 -0600
+++ src/wyrd-1.4.4/Makefile.in	2008-02-23 21:24:29.187500000 -0600
@@ -119,19 +119,19 @@
 
 install: install-indep
 	install -m 755 $(NAME).$(OCAMLBEST) $(BINDIR)
-	mv $(BINDIR)/$(NAME).$(OCAMLBEST) $(BINDIR)/$(NAME)
+	mv $(BINDIR)/$(NAME).$(OCAMLBEST) $(BINDIR)/$(NAME)$(EXE)
 
 install-byte: install-indep
 	install -m 755 $(NAME).byte $(BINDIR)
-	mv $(BINDIR)/$(NAME).byte $(BINDIR)/$(NAME)
+	mv $(BINDIR)/$(NAME).byte $(BINDIR)/$(NAME)$(EXE)
 
 install-opt: install-indep
 	install -m 755 $(NAME).opt $(BINDIR)
-	mv $(BINDIR)/$(NAME).opt $(BINDIR)/$(NAME)
+	mv $(BINDIR)/$(NAME).opt $(BINDIR)/$(NAME)$(EXE)
 
 install-static: install-indep
 	install -m 755 $(NAME).static-opt $(BINDIR)
-	mv $(BINDIR)/$(NAME).static-opt $(BINDIR)/$(NAME)
+	mv $(BINDIR)/$(NAME).static-opt $(BINDIR)/$(NAME)$(EXE)
 
 uninstall:
 	rm -f $(BINDIR)/$(NAME)$(EXE)
@@ -159,7 +159,7 @@
 CURSES_INCLUDES = -I ./curses
 CURSES_BFLAGS = -g $(CURSES_INCLUDES)
 CURSES_OFLAGS = $(CURSES_INCLUDES)
-CURSES_CFLAGS = $(CPPFLAGS) $(DEFS) $(CFLAGS) -Wall -fPIC -DPIC
+CURSES_CFLAGS = $(CPPFLAGS) $(DEFS) $(CFLAGS) -Wall -DPIC
 
 curses/%.cmi : curses/%.mli
 	$(OCAMLC) -c $(CURSES_BFLAGS) $<
--- origsrc/wyrd-1.4.4/configure.in	2008-02-21 22:25:25.000000000 -0600
+++ src/wyrd-1.4.4/configure.in	2008-02-23 21:24:29.203125000 -0600
@@ -261,7 +261,7 @@
 
 
 # check Remind version
-AC_PATH_PROG(REMINDPATH, remind, not found)
+AC_PATH_PROG(REMINDPATH, remind$EXEEXT, not found)
 if test x"$REMINDPATH" != x"not found"; then
    AC_MSG_CHECKING(remind version)
    REMINDVERSION=`strings $REMINDPATH | grep 03\.0`
